x/ccv/consumer/keeper/params.go

```diff
@@ -62,9 +65,28 @@ func (k Keeper) SetProviderFeePoolAddrStr(ctx sdk.Context, addr string) {
 	k.paramStore.Set(ctx, types.KeyProviderFeePoolAddrStr, addr)
 }
 
-// GetCCVTimeoutPeriod returns the timeout period for sent ibc packets
+// GetCCVTimeoutPeriod returns the timeout period for sent ccv related ibc packets
 func (k Keeper) GetCCVTimeoutPeriod(ctx sdk.Context) time.Duration {
 	var p time.Duration
 	k.paramStore.Get(ctx, ccvtypes.KeyCCVTimeoutPeriod, &p)
 	return p
 }
+
+// GetTransferTimeoutPeriod returns the timeout period for sent transfer related ibc packets
```

Comment:
Should setters exist for all of these parameters? Or only the ability to set their value on genesis?

If setters should exist, I'll need to add to https://github.com/cosmos/interchain-security/blob/more-params-rebase/x/ccv/consumer/keeper/params_test.go#L15